tests/reconfiguration-tests.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+
+setup_file() {
+	run kubectl --context $CLUSTER_CONTEXT delete --wait --timeout $TIMEOUT --ignore-not-found clusteradmissionpolicies --all
+	run kubectl --context $CLUSTER_CONTEXT delete --wait --timeout $TIMEOUT --ignore-not-found -f $RESOURCES_DIR/violate-privileged-pod-policy.yaml
+	run kubectl --context $CLUSTER_CONTEXT delete --wait --timeout $TIMEOUT --ignore-not-found -f $RESOURCES_DIR/not-violate-privileged-pod-policy.yaml
+	run kubectl --context $CLUSTER_CONTEXT wait --timeout $TIMEOUT --for=condition=Ready --namespace $NAMESPACE pods --all
+}
+
+@test "[Reconfiguration tests] Test apply policy" {
+	run kubectl --context $CLUSTER_CONTEXT apply --wait --timeout $TIMEOUT -f $RESOURCES_DIR/privileged-pod-policy.yaml --wait --timeout $TIMEOUT
+	[ "$status" -eq 0 ]
+	run kubectl --context $CLUSTER_CONTEXT wait --timeout $TIMEOUT --for=condition=PolicyActive clusteradmissionpolicies --all
```

Comment:
I think it would be good to abstract this one to a function that we call in the rest of the instances, also across test files, short of a bash utility function file that we import in the different test files. It makes reading the test shorter and to the point.

Reply:
argh, I failed to read this message prior to this getting merged. I'm concerned that we are going to end up with a lot of env vars and functions, and not be able to just copy-paste and run 1 line of the tests manually and easily when developing. I confess that I pictured the tests way simpler, just a bunch of commands to execute, and little more. Maybe I failed to see what it entailed before starting, or I'm having PTSD from bash-based testsuites ([mkcloud](https://github.com/SUSE-Cloud/automation/tree/master/scripts), [catapult](https://github.com/SUSE/catapult))

Maybe it's a silly concern, but I felt like writing it here.